src/test/test_random_walk.py

```diff
@@ -21,25 +22,16 @@ def test_rw_can_be_sampled():
     )
     init_rv_fixed = DeterministicVariable(name="init_rv_fixed", value=50.0)
 
-    step_rv = DistributionalRV(
-        name="rw_step",
-        distribution=dist.Normal(0, 1),
-    )
-
-    rw_init_rand = SimpleRandomWalkProcess(
-        "rw_rand_init", step_rv=step_rv, init_rv=init_rv_rand
-    )
-
-    rw_init_fixed = SimpleRandomWalkProcess(
-        "rw_fixed_init", step_rv=step_rv, init_rv=init_rv_fixed
+    rw = RandomWalk(
+        "rw",
+        DistributionalRV(name="rw_step_rv", distribution=dist.Normal(0, 0.27)),
     )
 
     with numpyro.handlers.seed(rng_seed=62):
         # can sample with a fixed init
         # and with a random init
-        ans_rand = rw_init_rand(n_steps=3532)
-        ans_fixed = rw_init_fixed(n_steps=5023)
-
+        ans_rand = rw(n=3532, init_vals=init_rv_rand()[0].value)
```

Comment:
I think the aspects of this test which rely on a random init are unnecessary. Isn't it really a test of whether `init_rv_rand()[0].value` is the size?

Reply:
Fair. This was a holdover from a previous implementation of `RandomWalk` that optionally sampled inits automagically. Can remove.

Reply:
Addressed in 75f14fb